Src/FluentAssertions/AggregateExceptionExtractor.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions.Common;
+using FluentAssertions.Specialized;
+
+namespace FluentAssertions
+{
+    public class AggregateExceptionExtractor : IExtractExceptions
```

Comment:
Should we make this `internal`?

Reply:
We can only do that if we give the specs access to its internals using `[InternalsVisibleTo`